scripts/skip-doc-change.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash -e
+#
+
+GIT_SINCE="${1}"
+if [ -z "${GIT_SINCE}" ]; then
+	GIT_SINCE='origin/master'
+fi
+
+CHANGED_FILES=$(git diff --name-only "${GIT_SINCE}")
+
+[[ -z $CHANGED_FILES ]] && exit 1
+
+skip=0
+#files to be skipped
+declare -a FILES=(^docs/ .md$ ^scripts/ LICENSE .mergify.yml .github .gitignore)
+
+function check_file_present() {
+    local file=$1
+    for FILE in "${FILES[@]}"; do
+        if [[ $file =~ $FILE ]]; then
```

Comment:
There are few others files are present in script folder will this check be enough?

Reply:
This script is a copy from the original/master `scripts/skip-doc-change.sh` with minor changes. If it needs to be adapted, please file an issue with your concerns.